app/assets/stylesheets/_top-bar.scss

```diff
@@ -0,0 +1,152 @@
+.top-bar {
+  position: relative;
+  padding: 12px 10px;
+  margin-bottom: 15px;
+  height: 52px;
+  font-size: 1.0em;
+  line-height: 140%;
+  color: $titlebar-text;
+  background-color: $white;
+  border-bottom: 1px solid $lightgrey-1;
+
+  &-logo {
+    position: relative;
+    top: -4px;
+    margin-right: 10px;
+  }
+
+  &-title {
+    font-size: 1.5em;
+    margin: 0.3em 0;
+  }
+
+  .project-title {
+    text-decoration: none;
+    font-weight: 600;
+    color: $darkgrey-2;
+    transition: 0.3s ease color;
+
+    &:hover {
+      color: $darkred-1;
+    }
+  }
+
+  .right-menu {
+    float: right;
+
+    > .dropdown {
+      display: inline;
+    }
+
+    button {
+      text-decoration: none;
+      font-weight: 600;
+      color: $darkgrey-2;
+
+      &:focus {
+        outline: none;
+      }
+    }
+
+    #form_search, .form_locale {
+      display: inline;
+
+      select {
+        color: $darkgrey-2;
+        font-weight: 600;
+        font-size: 14px;
+      }
+
+      .form-control {
+        width: auto;
+        display: inline;
+      }
+
+      .btn-group {
+        width: auto;
+
+        button {
+          background: transparent;
+          border: none;
+          box-shadow: none;
+          width: auto;
+
+          &:active {
+            box-shadow: none;
+          }
+
+          &:focus, span:focus {
+            outline: none;
+          }
+        }
+      }
+    }
+
+    .locale-change {
+      border: 0;
+      box-shadow: none;
+      background-color: transparent;
+    }
+  }
+
+  div.velocity {
+    position: relative;
+    display: inline;
+    font-size: 1.2em;
+
+    div.velocity_override_container {
+      width: 220px;
+      position: absolute;
+      left: 0;
+      top: 30px;
+      background: #efefef;
+      border: 1px solid #ccc;
+      @include border-radius(3px);
```

Comment:
I think it's better to keep all mixins on the top of the rule set, what do you think?